repository/managers/citesphere.py

```diff
@@ -31,6 +31,25 @@ def group_items(self, group_id, page=1):
             f'{self.endpoint}/api/v1/groups/{group_id}/items',
             params=params
         )
+        
+    def group_item(self, group_id, item_id):
+        return self._get_response(
+            f'{self.endpoint}/api/v1/groups/{group_id}/items/{item_id}'
+        )
+        
+    def content(self, content_id, group_id=None, item_id=None):
+        with open('/home/sudheera/dev/django/vogon-web/repository/managers/data','r') as file:
```

Comment:
If your going to include this, it needs to be a relative path or it needs to be env variable. This path will only work on your machine